src/elements/ProgressBar/ProgressBar.styl

```diff
@@ -84,13 +84,18 @@
 
     border none
     border-radius var(--progress-bar-border-radius)
+    color var(--progress-bar-bar-color)
```

Comment:
It seems that this line has no effect.

Reply:
this line is needed for IE and Edge. Remove it and Progress bar wont work for these browsers

Reply:
Okay :+1: